woocommerce/woocommerce
# [dev] Remove the unnecessary dependency for validate changelog and markdown jobs

### Changes proposed in this Pull Request:

`validate-changelog` and `validate-markdown` do not need to wait for `project-jobs` and can run immediately and in parallel.

### How to test the changes in this Pull Request:

Check CI. Tested with [this](https://github.com/woocommerce/woocommerce/actions/runs/12317617869/job/34380684638?pr=53703) test run where the jobs ran in parallel.
